src/components/cards/BaseStakingCard.tsx

```diff
@@ -650,8 +631,7 @@ const BaseStakingCard: FC<IBaseStakingCardProps> = ({
 									</FlexCenter>
 								</StakeContainer>
 							</StakeButtonsRow>
-							{active &&
-								!archived &&
+							{(exploited || isDiscontinued) &&
```

Comment:
It seems the condition must be contradicted
```suggestion
							{!(exploited || isDiscontinued) &&
```